_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -0,0 +1,183 @@
+    proposes an equal number of HTLC changes).  Responsibility for
+    proposing changes can be transferred from one party to the other as
+    needed, allowing each party to propose channel updates at different
+    times.
+
+    As of this writing, the proposal received only a small amount of
+    discussion from other LN implementation maintainers, so its future
+    remains uncertain.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19953][] implements [schnorr signature
+  verification][topic schnorr signatures] ([BIP340][]), [taproot][topic
+  taproot] verification ([BIP341][]), and [tapscript][topic tapscript]
+  verification ([BIP342][]).  The new soft fork rules are currently only
+  used in the private regression test mode (regtest); they are not
+  enforced in mainnet, testnet, or the default signet.  The plan is to
+  release the code in Bitcoin Core 0.21.0 and then prepare a subsequent
+  release (e.g. 0.21.1) that can begin enforcing the soft fork's new
+  rules when an activation signal is detected.  The particular
+  activation signal or signals to use are still being discussed in the
+  [##taproot-activation][] IRC chatroom ([logs][##taproot-activation
+  logs]).
+
+    {% comment %}<!-- $ git diff 450d2b2371...865d2c37e2 | grep '^+' \
+                        | grep -v '^+$' | grep -v '^+ */[/*]' | grep -v '^+ *\*' \
+                        | grep -v '^+++' | wc -l
+                      512
+    -->{% endcomment %}
+
+    The code [consists][sipa summary] of about [700 lines][sipa
+    consensus] of consensus-related changes (500 excluding comments and
+    whitespace) and [2,100 lines][sipa tests] of tests.  Over 30 people
+    directly reviewed this PR and its [predecessor][Bitcoin Core
+    #17977], and many others participated in developing and reviewing
+    the underlying research, the BIPs, the related code in libsecp256k1,
+    and other parts of the system.  There is still more work to do, but
+    getting the code properly reviewed was arguably the most critical
+    step---so we extend our most grateful appreciation to everyone who
+    helped achieve this milestone.
+
+- [Bitcoin Core #19988][] refactors the logic Bitcoin Core uses to
```

Comment:
s/refactors/redesigns/

'refactors' is too weak here. We usually use that to mean changes to the code that don't alter the logic or functionality. This PR was a complete redesign of the way we select which peers to download transactions from.